doc/content/integrations/cloud-integrations/blockbax/_index.md

```diff
@@ -19,133 +19,98 @@ Check the [Blockbax documentation page](https://blockbax.com/docs/) for more inf
+Create an [inbound connector](https://blockbax.com/docs/project-settings/#inbound-connectors) accepting HTTP JSON payloads. The payload conversion script depends on the way the payload is formatted before it is being sent to Blockbax. By default when you add a known device in {{% tts %}} it comes with a payload formatter submitted by the device manufacturer from the [device repository]({{< ref "/integrations/payload-formatters/device-repo" >}}). In most cases data is returned as a simple JavaScript object containing numeric and text values as properties.
 
-{{< figure src="blockbax-metric-external-id-in-payload.png" alt="Relating Blockbax metrics to the Things Stack" >}}
+In this case everything will work with the simple conversion script below. If not, you need to align the payload conversion script to the way your device formatter(s) output the payload. We will explain how to do this later. In any case it is a good starting point to start with the following script in your inbound connector.
 
-## Configure {{% tts %}}
+```javascript
+/**
+ * @param {JsonPayload} payload
+ * @param {Context} context
+ */
+function convertPayload(payload, context) {
+    const timestamp = date(payload.received_at);
+    const subjectExternalId = payload.end_device_ids.device_id;
+    for (const [metricExternalId, value] of Object.entries(payload.uplink_message.decoded_payload)) {
+        if (value != null) {
+            context.addMeasurement(subjectExternalId + "$" + metricExternalId, value, timestamp);
+        }
+    }
+}
+```
 
-Make sure your device is added to {{% tts %}}. See [Adding Devices]({{< ref "/devices/adding-devices" >}}) for more info. Many [Device Repository]({{< ref "/integrations/payload-formatters/device-repo" >}}) payload formatters output payloads such that it can be directly ingested by the Blockbax Platform. If this is not the case, then please follow the instruction below to create a custom payload formatter.
+Optionally you can check 'Automatically create subjects'. This will create a subject automatically if a subject does not exist.
 
-### Create payload formatter
+## Create a webhook in {{% tts %}}
 
-On the left menu of your {{% tts %}} application, select **Payload formatters &#8594; Uplink**.
+Use the Blockbax [Webhook template]({{< ref "/integrations/webhooks/webhook-templates" >}}) to create a Webhook integration on {{% tts %}}. Select **Integrations &#8594; Webhooks** on the left hand menu. Click **Add webhook** and select the **Blockbax** tile.
 
-Select **Javascript** payload formatter type.
+{{< figure src="webhook.png" alt="Setting up the webhook" >}}
 
-Paste the following contents in the **Formatter parameter** window:
+Enter an arbitrary **Webhook ID**, enter your **[Access token](https://blockbax.com/docs/project-settings/#access-tokens)** and **Endpoint**. The endpoint you can copy from the inbound connector settings.
 
```

Comment:
Since the baseURL of the webhook template has to be absolute (as I mentioned [here](https://github.com/TheThingsNetwork/lorawan-webhook-templates/pull/67)), the user won't be able to copy the whole endpoint from Blockbax and just paste it in the TTS webhook template. See [this](http://localhost:1313/integrations/webhooks/webhook-templates/instantiation/#instantiation-of-urls-and-paths) for an example.

Reply:
Our user copies an absolute URL from the Blockbax Platform and pastes it in the 'Endpoint' field in TTS. I can imagine it wasn't what you expected so to clarify the picture below of the URL the user copies. See the 'Copy' in the field below 'Endpoint'. Would it work if it would past this full / absolute URL? 

![image](https://user-images.githubusercontent.com/34166358/184220223-111393c8-7e7f-4d9e-9282-06a42fd476dc.png)

See also my comment one the webhook template [here](https://github.com/TheThingsNetwork/lorawan-webhook-templates/pull/67#issuecomment-1212386669).

Reply:
> Our user copies an absolute URL from the Blockbax Platform and pastes it in the 'Endpoint' field in TTS. I can imagine it wasn't what you expected so below a screenshot of how it looks to our users where they copy the URL. See the 'Copy' in the field below 'Endpoint'. Would it work if the user would paste this full / absolute URL?
> 
> ![image](https://user-images.githubusercontent.com/34166358/184220223-111393c8-7e7f-4d9e-9282-06a42fd476dc.png)
> 
> See also my comment one the webhook template [here](https://github.com/TheThingsNetwork/lorawan-webhook-templates/pull/67#issuecomment-1212386669).

I see why this is the most straightforward and user friendly way to do this, but I think TTS has certain baseURL limitations that don't allow this. Let's first conclude the discussion on the webhook template issue and then you can fix this if needed.

Reply:
yep, ofc 😊